docs/tutorials/intro_tutorial.rst

```diff
@@ -879,273 +895,208 @@ True indefinitely.
                 x = self.random.randrange(self.grid.width)
                 y = self.random.randrange(self.grid.height)
                 self.grid.place_agent(a, (x, y))
-
+    
             self.datacollector = DataCollector(
-                model_reporters={"Gini": compute_gini},
-                agent_reporters={"Wealth": "wealth"})
-
+                model_reporters={"Gini": compute_gini}, agent_reporters={"Wealth": "wealth"}
+            )
+    
         def step(self):
             self.datacollector.collect(self)
             self.schedule.step()
 
-We instantiate a BatchRunner with a model class to run, and two
-dictionaries: one of the fixed parameters (mapping model arguments to
-values) and one of varying parameters (mapping each parameter name to a
-sequence of values for it to take). The BatchRunner also takes an
-argument for how many model instantiations to create and run at each
-combination of parameter values, and how many steps to run each
-instantiation for. Finally, like the DataCollector, it takes
-dictionaries of model- and agent-level reporters to collect. Unlike the
-DataCollector, it won't collect the data every step of the model, but
-only at the end of each run.
+We call ``batch_run`` with a model class to run, and a dictionary with
+the model parameters where each of the values can either be fixed or an
```

Comment:
I think we need to elaborate on how to specify the parameters in the case of fixed vs iterable.

Reply:
The older documentation already explains it succinctly, because of the clear separation between fixed and varying parameters.

Reply:
I agree as this often becomes a sticky point for students... just a thought on how to succinctly convey and I by no means tied to this. 

_We call ``batch_run`` with a model class to run, and a dictionary with the model parameters where each of the values can either be fixed ( e.g. {"height": 10, "width": 10}) or an iterable  (e.g. "num_agents": range(10, 500, 10) )_